activation/poetlistener.go

```diff
@@ -41,7 +37,11 @@ func (l *PoetListener) HandlePoetProofMessage(ctx context.Context, _ p2p.Peer, m
 		return pubsub.ValidationIgnore
 	}
 
-	if err := l.poetDb.storeProof(&proofMessage); err != nil {
+	if err := l.poetDb.StoreProof(&proofMessage); err != nil {
+		if err == sql.ErrObjectExists {
+			// don't spam the network
+			return pubsub.ValidationIgnore
+		}
```

Comment:
i thought that we are already not sending it to the network in cases when validation failed :/.

what about layerfetcher? will it handle ErrObjectExists correctly?

Reply:
it treated it as a validation error which i think incorrect. fixed